vignettes/rebuild_cached_data.Rmd

```diff
@@ -42,17 +43,10 @@ RNGkind(sample.kind = "Rounding")
 
 # Datasets must be listed after all of their dependencies
 # e.g. adsl is a dependency for all other datasets so it is listed first.
```

Comment:
@Melkiades I reverted the auto dataset name listing because that does not take into account dependency orders (see comment in the code) which means, for example if both `adpc` and `adab` are updated then `cadab` will be updated first since it is first alphabetically even though it should be updated after its dependency `cadpc`. 

I don't think it will be difficult to maintain a manual list (only needs to be modified when a new dataset is added). 

Let me know what you think!

Reply:
ok now I understand why you did so. I am working on a more automatic solution that takes the order into account correctly. The problem is taking a tree and transforming it in a list of deps